lib/Services/Request.ts

```diff
@@ -0,0 +1,14 @@
+import { Context, Data, Effect } from "effect";
+
+export class RequestJsonError extends Data.TaggedError("RequestJsonError")<{
+  error: unknown;
+}> {}
+
+export const Request = Context.Tag<Request>("Request");
+
+export const requestJson = Request.pipe(
```

Comment:
Because we moved Request to Context, we can now create re-usable helper functions that work with it.